lib/programmemory.cpp

```diff
@@ -709,133 +709,133 @@ static std::unordered_map<std::string, BuiltinLibraryFunction> createBuiltinLibr
         return v;
     };
     functions["atan2"] = [](const std::vector<ValueFlow::Value>& args) {
-        if (args.size() != 2)
-            return ValueFlow::Value::unknown();
-        ValueFlow::Value v = args[0];
```

Comment:
This line here is to propagate the other properties of the value(ie errorPath, conditional, path, etc). For binary functions we should probably use combineValueProperties instead.